### PR TITLE
[AMS-2018] add organiser

### PR DESCRIPTION
**Pull request by Peter-Nijenhuis**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2018] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: data/events/2018-amsterdam.yml [hunk 1]
```diff
@@ -75,6 +75,8 @@ team_members: # Name is the only required field for team members.
     twitter: "cdlennon"
   - name: "Daniel Paulus"
     twitter: "dpnl87"
+     - name: "Greg Trahair"
```
Comment (dpnl87) [inline data/events/2018-amsterdam.yml]: The problem is here. Please align with the above '-'

**Comment by phrawzty** [comment]
Hello @Peter-Nijenhuis. I fixed the title of this PR as well as an error in the YAML syntax.

Before this can be merged, please send an email to info@devopsdays.org with the full name and email address of this organiser, so that both Slack and the AMS 2018 mailing list can be updated.

**Comment by dpnl87** [comment]
@phrawzty you are super fast! =)

**Comment by Peter-Nijenhuis** [comment]
HI Daniel,

the email with full name and email address of the organizer has been send to info@devopsdays.org (https://link.getmailspring.com/link/1530877872.local-03f91341-4ae7-v1.2.2-96fb3a99@getmailspring.com/0?redirect=mailto%3Ainfo%40devopsdays.org&recipient=cmVwbHkrMDEzNjE3MjM5MGRkMjAwNGRhNWI5OWI3YjJhMzFlYjY1NWRmMTIyMjMyZjczNmNiOTJjZjAwMDAwMDAxMTc1NzE5N2M5MmExNjljZTE0MzM1MzNiQHJlcGx5LmdpdGh1Yi5jb20%3D)
Cheers,

Met vriendelijke groeten / Kind regards

**Comment by phrawzty** [comment]
Slack invite sent, mailing list updated, and PR about to be merged. :)

**Comment by Peter-Nijenhuis** [comment]
thanks!

**Pull request merged**